com.unity.ml-agents/Runtime/Demonstrations/DemonstrationRecorder.cs

```diff
@@ -32,6 +32,11 @@ public class DemonstrationRecorder : MonoBehaviour
         [Tooltip("Whether or not to record demonstrations.")]
         public bool Record;
 
+        /// <summary>
+        /// Number of steps to record. Editor will stop playing when it hits this threshold.
+        /// </summary>
+        public int numStepsToRecord = 5000;
```

Comment:
```suggestion
        public int NumStepsToRecord = 5000;
```
We're using PascalCase for public fields and properties.

Reply:
changed to PascalCase